idaes/core/components.py

```diff
@@ -52,10 +52,22 @@ class ComponentData(ProcessBlockData):
             doc="Dict indicating phases in which component follows Henry's "
                 "Law (keys) with values indicating form of law."))
 
+    CONFIG.declare("vol_mol_liq_comp", ConfigValue(
+        description="Method to use to calculate liquid phase molar volume",
+        doc="Method to use to calculate liquid phase molar volume. Users "
+        "need only define one of vol_mol_liq_comp and dens_mol_liq_comp."))
```

Comment:
Do you mean that users only need to define either (a) molar volume or (b) molar density, but not both?

Reply:
Yes - dens_mol = 1/vol_mol, so you only need one of them. Do you think this could be more clear (suggestions welcome, but that might also be something better suited to the docs rather than just a doc string).